dist/pathfora.js

```diff
@@ -2292,6 +2291,118 @@
 
   /**
    * @class
+   * @name Inline
+   * @description Inline Personalization
+   */
+  Inline = function () {
+    this.elements = [];
+    this.preppedElements = [];
+
+    /*
+     * @description Prepare all the elements with the attribute provided
+     * @param {attr} name of attribute to select by
+     */
+    this.prepElements = function (attr) {
+      var dataElements = {};
+
+      if (this.elements.length === 0) {
+        this.elements = document.querySelectorAll('[' + attr + ']');
+      }
+
+      var elements = this.elements;
+
+      for (var i = 0; i < elements.length; i++) {
+        if (elements[i].getAttribute(attr) !== null) {
+          var theElement = elements[i],
+              group = theElement.getAttribute('data-liogroup'),
+              type = theElement.getAttribute('data-liotype'),
```

Comment:
totally minor and your call but would it be better to rename all these attributes now that they are outside of lio? can probably do something a bit more clean like:

`pfa-group`
`pfa-type`
etc?